docs/launch_modes.md

```diff
@@ -70,3 +86,12 @@ Triton Inference Server.
    Analyzer does not currently support profiling remote GPUs. Triton Server in this
    mode needs to be launched with `--model-control-mode explicit` flag to support
    loading/unloading of the models.
+
+### Tradeoffs
```

Comment:
Instead of using "tradeoffs" which can imply lack of features, how about renaming this section to "Deciding what mode to use?" or similar? Or we could remove this section and put a sentence in the end of the description for each mode on when this mode is useful? For example:

Remote mode is useful when you have an already running instance of Triton that you cannot restart. The model config parameters cannot be changed in remote mode.

Reply:
removed this section and put a sentence at the end of the description for each mode on when that mode is useful 👍